### PR TITLE
Add CHANGES and NEWS for 1.1.1c

### PR DESCRIPTION
**Pull request by levitte**

### CODE REVIEW

**Comment by rainerjung** [comment]
Trivial typo fix in the above CHANGES patch: "cipher ssuite" => "cipher suite"

**Review by mattcaswell — APPROVED** [review]
Approved subject to the typo spotted by @rainerjung being fixed.

File: CHANGES [hunk 1]
```diff
@@ -9,20 +9,75 @@
...
+  *) Reorganize the manual pages to consistently have RETURN VALUES,
+     EXAMPLES, SEE ALSO and HISTORY come in that order, and adjust
+     util/fix-doc-nits accordingly.
+     [Paul Yang, Joshua Lock]
+
+  *) Add the missing accessor EVP_PKEY_get0_engine()
+     [Matt Caswell]
+
+  *) Have apps like 's_client' and 's_server' output the signature scheme
+     along with other cipher ssuite parameters when debugging.
```
Comment (beldmit) [inline CHANGES]: Typo. ssuite => suite.

Comment (levitte) [reply CHANGES]: Fixed

**Comment by levitte** [comment]
Merged.

3e3f4e903b4ecc8f271d591010ee5609211ac545 Add CHANGES and NEWS for 1.1.1c

**Pull request closed without merging**